ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -235,12 +242,14 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeMaxNumberOfEntriesToReturnFromJournal =
         verifiableProperties.getIntInRange("store.max.number.of.entries.to.return.from.journal", 5000, 1, 10000);
     storeDeletedMessageRetentionDays = verifiableProperties.getInt("store.deleted.message.retention.days", 7);
+    storeHardDeleteOperationsBytesPerSec =
```

Comment:
The `Math.min()` within the `HardDeleter` constructor is effectively capping this at `1024 * 1024`, so the default you set here has no meaning. Neither does the range of `(0, Integer.MAX_VALUE)` have a meaning.

Is `1024 * 1024` really the max that we want to go with? If so, enforce it here in `StoreConfig` rather than in the `HardDeleter` constructor.

Reply:
Agree. We don't need a min. This value can be enforced in storeConfig. 
For HardDelete, the default value is 1MB and range is (1024*1024, INT.MAX).